gulpfile.js

```diff
@@ -30,6 +30,12 @@ gulp.task('copy-vendors', function () {
 		.pipe(gulp.dest('./public/js'));
 });
 
+gulp.task('copy-assets', function () {
```

Comment:
Issue is not about crazy, it's about establishing gulp task patterns to keep concurrency maximised, and therefore the developer's workflow as fast as possible, which is a stated objective.